README.md

```diff
@@ -1 +1,23 @@
-# portfolio
\ No newline at end of file
+# Project Name:portfolio
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
```

Comment:
- [ ] Please write the steps on how a user can get to use this project 👍  like how one can set up this project on their local machine 💻